formulaire-public/src/main/resources/public/ts/directives/public-question-item.ts

```diff
@@ -97,6 +99,14 @@ class Controller implements IViewModel {
         }
         else return;
     }
+
+    onClickChoice = (choice: QuestionChoice) : void => {
+        this.responses.all[0].choice_id = (this.responses.all[0].choice_id != choice.id) ? choice.id : null;
```

Comment:
On est d'accord qu'avec ce life hook `$scope.on...` (cf https://github.com/OPEN-ENT-NG/form/blob/1cb3efa063201dd8e6ade1ca2af47b18d897b872/formulaire-public/src/main/resources/public/ts/directives/public-question-item.ts#L48)

L'instruction `this.responses = data.get(this.question);` te permettra toujours d'avoir un tableau avec au moins un élement ?

Pour éviter un potentiel `cannot read property of null`.

Reply:
C'est ça. L'idée c'est qu'on a des objets Response vident et on travaille dessus en remplissant leur champs "answer", "choice_id" ou autre